makepdf

```diff
@@ -117,13 +117,12 @@ while [ "${*}" != "" ] ; do
         if [[ ! -s "${TMP_PNG_DIR}/$(basename "${TIF%.*}").png" ]] ; then
             ffmpeg -i "${TIF}" -pix_fmt gray -y "${TMP_PNG_DIR}/$(basename "${TIF%.*}").png"
         fi
-        convert "${TMP_PNG_DIR}/$(basename "${TIF%.*}").png" -level 10%,75%,1 -blur 2 +dither -resize 2550x3300 -monochrome -flatten +repage -define png:bit-depth=8 -define png:color-type=0 "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png"
-        tesseract "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png" "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}")" -l eng -psm 4 pdf "${SCRIPTDIR}/tesseract.conf"
-        tesseract "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png" "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}")" -l eng -psm 4 "${SCRIPTDIR}/tesseract.conf"
-    done
+        convert "${TMP_PNG_DIR}/$(basename "${TIF%.*}").png" -blur 2 +dither -resize 1545x2000 -flatten +repage -define png:bit-depth=8 -define png:color-type=0 "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png"
+        tesseract "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png" "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}")" -l eng --psm 4 pdf "${SCRIPTDIR}/tesseract.conf"
+        tesseract "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png" "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}")" -l eng --psm 4 "${SCRIPTDIR}/tesseract.conf"
```

Comment:
I suggest leaving both the old way and new way together to get all outputs until we ok what those outputs are.
We had good success getting a jpeg from ffmpeg directly into tesseract and skipping convert for non-bw outputs.  Such as:

```
ffmpeg -i HBS03554_5.tif -s 2550x3300 HBS03554_5new.jpg
tesseract HBS03554_5new.jpg HBS03554_5new_jpg -l eng --psm 4 pdf
tesseract HBS03554_5new.jpg HBS03554_5new_jpg -l eng --psm 4
```

Reply:
Also we had good luck with 1/16 size images (ie 5100/4 x 6600/4 rather than 5100/2 x 6600/2 as used in the example above). Possibly if this method of using smaller jpegs rather than b/w pngs is ok, it could be that we drop the convert command and use of b/w images altogether.

Reply:
we didn't get to figuring out how to produce all three at once due to the problems i mentioned below... since i think i figured out the source of that problem i will work on this next week